packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -2014,11 +2009,9 @@ class _CupertinoTimerPickerState extends State<CupertinoTimerPicker> {
     final double centerPoint = paddingStart + (numberLabelWidth / 2);
 
     // Compute the offAxisFraction needed to be straight within the pickerColumn.
-    final double pickerColumnOffAxisFraction =
-        0.5 - centerPoint / pickerColumnWidth;
+    final double pickerColumnOffAxisFraction = 0.5 - centerPoint / pickerColumnWidth;
```

Comment:
I'm not really sure that I feel strongly that the left is properly formatted, but I'm not sure if this is any better.

its odd to split up the expression itself, instead of on the assignment